src/quacc/runners/ase.py

```diff
@@ -362,3 +369,171 @@ def _set_sella_kwargs(self, optimizer_kwargs: dict[str, Any]) -> None:
+
+    # Set defaults
+    optimizer_kwargs = recursive_dict_merge(
+        {
+            "logfile": "-" if SETTINGS.DEBUG else dir_lists[0][0] / "opt.log",
+            "restart": dir_lists[0][0] / "opt.json",
+        },
+        optimizer_kwargs,
+    )
+    run_kwargs = run_kwargs or {}
+
+    # Check if trajectory kwarg is specified
+    if "trajectory" in optimizer_kwargs:
+        msg = "Quacc does not support setting the `trajectory` kwarg."
+        raise ValueError(msg)
+
+    # Define the Trajectory object
+    traj_file = dir_lists[0][0] / "neb.traj"
+    traj = Trajectory(traj_file, "w", atoms=neb)
+    optimizer_kwargs["trajectory"] = traj
+
+    # Set volume relaxation constraints, if relevant
+    for i in range(len(images)):
+        if relax_cell and images[i].pbc.any():
+            images[i] = FrechetCellFilter(images[i])
+
+    # Run optimization
+    with traj, optimizer(neb, **optimizer_kwargs) as dyn:
+        dyn.run(fmax=fmax, steps=max_steps, **run_kwargs)
+
+    # Store the trajectory atoms
+    dyn.traj_atoms = read(traj_file, index=":")
+
+    # Perform cleanup operations
+    for ii, image in enumerate(images):
+        calc_cleanup(image, dir_lists[ii][0], dir_lists[ii][1])
+
+    return dyn
+
+
+def _geodesic_interpolate_wrapper(
+    reactant: Atoms,
+    product: Atoms,
+    nimages: int = 20,
+    perform_sweep: bool | None = None,
+    convergence_tolerance: float = 2e-3,
+    max_iterations: int = 15,
+    max_micro_iterations: int = 20,
+    morse_scaling: float = 1.7,
+    geometry_friction: float = 1e-2,
+    distance_cutoff: float = 3.0,
+) -> tuple[list[str], list[list[float]]]:
+    """
+    Interpolates between two geometries and optimizes the path with the geodesic method.
+
+    Parameters
+    ----------
+    reactant_product_atoms : List[Atoms]
+        List of ASE Atoms objects containing initial and final geometries.
+    nimages : int, optional
+        Number of images for interpolation. Default is 20.
+    perform_sweep : Optional[bool], optional
+        Whether to sweep across the path optimizing one image at a time.
+        Default is to perform sweeping updates if there are more than 35 atoms.
+    convergence_tolerance : float, optional
+        Convergence tolerance. Default is 2e-3.
+    max_iterations : int, optional
+        Maximum number of minimization iterations. Default is 15.
+    max_micro_iterations : int, optional
+        Maximum number of micro iterations for the sweeping algorithm. Default is 20.
+    morse_scaling : float, optional
+        Exponential parameter for the Morse potential. Default is 1.7.
+    geometry_friction : float, optional
+        Size of friction term used to prevent very large changes in geometry. Default is 1e-2.
+    distance_cutoff : float, optional
+        Cut-off value for the distance between a pair of atoms to be included in the coordinate system. Default is 3.0.
+
+    Returns:
+    --------
```

Comment:
Numpy-style docstrings do not have `Returns:` but rather just `Returns`. 

These and the other inconsistencies are why your docs test is failing in CI.

Reply:
Done.